xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxUtils.cpp

```diff
@@ -23,6 +23,7 @@
 #endif
 #include "DVDDemuxUtils.h"
 #include "DVDClock.h"
+#include "linux/XMemUtils.h"
```

Comment:
Oh, there was the only definition of _aligned functions I found, but I didn't know these were provided by the MSVC runtime.